src/components/graph/GraphZoom.tsx

```diff
@@ -12,27 +12,32 @@ interface Props {
 type zoomItem = {
 	value: IntervalOption;
 	text: string;
+	toggle: boolean;
 };
 
 const GraphZoomOptions = (props: Props) => {
 	const { setInterval, setTimestamp, windowRange } = props;
-	const zoomItems: zoomItem[] = [
-		{ value: 15, text: '1x' },
-		{ value: 60, text: '4x' },
-		{ value: 240, text: '16x' },
-	];
+	const [zoomItems, setZoomItems] = React.useState<Array<zoomItem>>([
+		{ value: 15, text: '1x', toggle: true },
+		{ value: 60, text: '4x', toggle: false },
+		{ value: 240, text: '16x', toggle: false },
+	]);
```

Comment:
Better to pass the interval prop to the component and add the active class, compare with value